### ISSUE TITLE
[TensorRT] ERROR: concatenate_1: all concat input tensors must have the same dimensions except on the concatenation axis (3), but dimensions mismatched at input 1 at index 1.

### ISSUE THREAD

**Issue by Neronjust2017** [NVIDIA/TensorRT#1324]
## Description

Hi, when i try to covert onnx (convert from keras) to tensorrt engine, I got this error:

[TensorRT] ERROR: concatenate_1: all concat input tensors must have the same dimensions except on the concatenation axis (3), but dimensions mismatched at input 1 at index 1. Input 0 shape: [1,8,8,256], Input 1 shape: [1,7,7,512]
[TensorRT] ERROR: concatenate_1: all concat input tensors must have the same dimensions except on the concatenation axis (3), but dimensions mismatched at input 1 at index 1. Input 0 shape: [1,8,8,256], Input 1 shape: [1,7,7,512]
[TensorRT] ERROR: Layer concatenate_1 failed validation
[TensorRT] ERROR: Network validation failed.

It seems Input 0 and Input 1 have different dimensions on axis(1) and (2). However, the dimensions are matched in the keras model and it can be trained successfully. Does anyone know what's the problem? Any suggestions to debug this? Thanks!

## Environment

**TensorRT Version**: 7.0.0.11
**NVIDIA GPU**: Tesla P100
**NVIDIA Driver Version**: 440.33.01
**CUDA Version**: 10.2
**CUDNN Version**: 7.6.5
**Operating System**: CentOS
**Python Version (if applicable)**: 3.6
**Tensorflow Version (if applicable)**: 1.14.0

>

**Comment by ttyio** [comment]
@Neronjust2017 , 
Could you take a try with latest 8.0? we have many fix there.
To debug if the model is correct, maybe you can try run ONNX on other tool like onnxruntime.
Thanks!

**Comment by Neronjust2017** [comment]
Hi, I found the problem. This is a problem caused by layer shape mismatching using images with small shape as the network input. It's not about TensorRT. Thanks.
